fuzzing/README.md

```diff
@@ -0,0 +1,29 @@
+# Fuzzing
+This folder contains a fuzzing setup that is meant to be used with OSS-Fuzz. 
+
+Fuzzing as a concept aims to stress-test the code under analysis. Traditional fuzzing is based on sending random input to the target application, whereas modern fuzzing relies on genetic algorithms based on code-coverage. The goal of the fuzzing in net-snmp is to catch any potential bugs, in particular memory corruption issues. The net-snmp code is instrumented with different sanitizers, e.g. AddressSanitizer, when being fuzzed. 
```

Comment:
All other Net-SNMP source code and README documents only use 80 columns. Should the width of this text be restricted to 80 columns too (except for hyperlinks)?

Reply:
I think they should - fixed it up now.